### PR TITLE
UI & Copy Content Fixes

### LINKED ISSUE #172: UI | Add new field for text transcription under detection images
## Description

As a compost bin user, I want to be able to see the text transcription collected by the bin during the compost process so that I can use that information to be reminded of what should be annotated.

### Why?
If we're going to have audio / voice transcriptions in the database, we should use it in the app!

## Solution/Requirements
* [ ] add new text transcription field directly below each image on the detection detail page
* [ ] add new text transcription field on the image card, directly above the image on the annotation page

## Dependencies
* [ ] update data payload being pulled from the database so that the text transcription data is available for use in the app

## Definition of Done
* [ ] the above two fields have been incorporated into the app UI

### PR DESCRIPTION
**Pull request by kimberlymarkley**
## Description
* add new field for text transcription under detection images
* add card styling to Annotation page adjust image and overlay size make button prompts more clear
* restyle detections list so images have borders, adjust padding for both list views (large/small)
* add card styling to Annotation page adjust image and overlay size make button prompts more clear
* restyle detections list so images have borders, adjust padding for both list views (large/small)
* adjust main page card padding
* bug fix for failure to load detections in the homepage graph visuals
* adjust padding around 'detections by food category' on home page
* center image in container, delete unnecessary imports
* delete statistic card widget, no longer in use after detection data field display refactor
* add doc comments for widget test pages
* change appearance of detection data fields from individual cards to list format add transcription field update associated db models and tests
* style error message displayed when there are no detections to display
* adjust image padding on single detection card, add border to match other pages tweak annotation button bottom padding
* add splash effect to annotation button and tooltip for accessibility
* adjust annotate button upwards to match its distance from right margin
* correct capitalization of binsight.ai app in drawer menu

## Related Issues
* References #101  
* Resolves #172 /

## Screenshots
Screenshot_1715797111
Screenshot_1715797120
Screenshot_1715797242
Screenshot_1715797248
Screenshot_1715970416
Screenshot_1715970569

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -210,7 +230,8 @@ class _BinsightAiAppState extends State<BinsightAiApp>
     //Defines the router to be used for the app, with set-up as the initial route
     setRoutes(getRoutes());
     router ??= GoRouter(
-        initialLocation: widget.skipSetUp ? '/main' : '/set-up',
+        // initialLocation: widget.skipSetUp ? '/main' : '/set-up',
+        initialLocation: widget.skipSetUp ? '/main' : '/main',
```
Comment (theusaf) [inline lib/main.dart:234]: Should be reverted

File: lib/pages/detection/annotation.dart [hunk 2]
```diff
@@ -98,13 +98,13 @@ class _AnnotationPageState extends State<AnnotationPage> {
...
                   decoration: BoxDecoration(
-                      border: Border.all(color: Colors.black, width: 2.0)),
+                      border: Border.all(color: Colors.black, width: 1.0)),
                   child: Image.asset('assets/images/annotation.gif'),
                 ),
                 Padding(
                   padding: const EdgeInsets.all(8.0),
                   child: Text(
-                      "Trace the composted item with your finger as accurately as possible.",
+                      "Outline composted item with your finger as accurately as possible.",
```
Comment (theusaf) [inline lib/pages/detection/annotation.dart:107]:
```suggestion
                      "Outline the composted item with your finger as accurately as possible.",
```

**Review by theusaf — CHANGES_REQUESTED** [review]
There are some commented code blocks and formatting issues that could be improved as well

**Review by stalberm — APPROVED** [review]

**Review by kimberlymarkley — COMMENTED** [review]
Thank you for the merge conflict solves, Daniel

**Review by theusaf — APPROVED** [review]

**Review by bpugh5 — APPROVED** [review]

**Pull request merged**